cmd/setup/setup.go

```diff
@@ -0,0 +1,128 @@
+package setup
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	ProdJumproleConfigKey  = "prod_jumprole_account_id"
+	AwsProxy               = "aws_proxy"
+	StageJumproleConfigKey = "stage_jumprole_account_id"
+	Pd_User_Token          = "pd_user_token"
+	Jira_Token             = "jira_token"
+	Dt_Vault_Path          = "dt_vault_path"
+	Vault_Address          = "vault_address"
```

Comment:
How can we make sure we don't forget adding new config parameters here? I think we should have a centralized place where the config file is defined and the setup command picks up the parameters from there. We could have that as a follow up card as well in case you think it goes beyond the issue you're currently working on :)

Reply:
I think this is still open, e.g. we missed a piece of config here, it didn't land in the setup: https://github.com/openshift/osdctl/blob/aa4dbcc47e6dc70555944070b81bb3e86d638033/pkg/envConfig/config.go#L81 

As mentioned, feel free to open a follow up card to have the whole configuration loading in a centralized place. E.g. one big `LoadConfig()` for all of osdctl. The setup command should re-use that to know what configurations should exist.

Reply:
As discussed, this can be covered in follow-up card.

Reply:
@typeid  I've added the CloudTrailList command in current PR and  config requirement will be covered in followup card for Common load config. I hope this looks good for as of now to close this PR.